### DISCUSSION TITLE
[Suggestion] Winget setting for non-silent installation

### DISCUSSION THREAD

**Discussion by yesseruser** [Devolutions/UniGetUI#2974] [General]
Since some people might need to see the installation progress, I am suggesting an option to enable/disable the silent flag.

**Comment by marticliment** [comment]
> Since using the silent flag could potentially break some packages' installation

Why do you say that? WinGet only silently installs packages that can be installed silently

**Comment by Lockszmith-GH** [comment]
You mean the 'install interactively'? - it's already there.

**Reply by yesseruser** [reply]
No, I meant the default mode when upgrading with the Winget command. It's non-interactive, but not silent. It's usually just a progress bar of the installation process.

**Reply by Lockszmith-GH** [reply]
I see, I was hoping winget has some default settings, but no.

I wonder if this can be a feature request for bundles. This way you can setup you own custom settings for your custom set of packages.

**Reply by Infinitay** [reply]
It would have been nice to have as default coming from someone who is transitioning to using WinGet via UniGetUI. I think that some of my apps that were previously installed via `.msi` setup files or just their default installers which were then instead updated/installed via WinGet were moved to a new location. Took me a while to realize this. For example, I had installed my `StreamLink` package originally via it's setup installer and installed it to a custom location on another drive. When updating it via UniGetUI I think it moved it to main OS drive instead of the prior installed location. I realized this some time after. Although, I will say that this behavior wasn't for every app. The only one that comes to mind is StreamLink's CLI app
